modules/auxiliary/scanner/http/http_version.rb

```diff
@@ -26,14 +26,28 @@ def initialize
     register_wmap_options({
         'OrderID' => 0,
         'Require' => {},
-      })
+    })
+
+    register_options([
+      OptString.new('TARGETURI', [true, 'The URI to use', '/'])
+    ])
+
+    register_advanced_options([
+      OptString.new('HTTP_METHOD', [true, 'HTTP Method to use', 'GET']),
```

Comment:
Actually, now that I said these should be advanced, I realized these are actually _evasion_ options and should be done with `register_evasion_options` instead.  

Also, FYI, these already exist and come close to what you are doing:

```
        OptBool.new('HTTP::method_random_valid', [false, 'Use a random, but valid, HTTP method for request', false]),
        OptBool.new('HTTP::method_random_invalid', [false, 'Use a random invalid, HTTP method for request', false]),
        OptBool.new('HTTP::method_random_case', [false, 'Use random casing for the HTTP method', false]),
```